src/Pos/Explorer/Txp/Toil/Logic.hs

```diff
@@ -95,8 +96,9 @@ eProcessTx tx@(id, aux) extra = do
     undo <- Txp.processTx tx
     putTxExtra id extra
     putNewHistory id $ getTxRelatedAddrs aux undo
-    -- let balanceUpdate = getBalanceUpdate aux undo
-    -- updateAddrBalances balanceUpdate
+    let balanceUpdate = getBalanceUpdate aux undo
+    fmap fst $ usingLoggerName "eProcessTx" $ runNamedPureLog $
```

Comment:
Here log events are discarded. Please add a TODO and also it makes sense to create a minor issue about it (in this case put issue identifier into TODO).